### PR TITLE
Tatar Translation!

### PR DESCRIPTION
**Pull request by Amirhan-Taipovjan-Greatest-I**
Vanilla-ish Tatar Translation.

### CODE REVIEW

**Review by TheCSDev — APPROVED** [review]

**Comment by TheCSDev** [comment]
Hello!
Thanks for the Tatar translation. I'm not sure when I'll next work on Forge, but if I ever do, I'll include this translation as well. Have a great day!

**Comment by Amirhan-Taipovjan-Greatest-I** [comment]
> Hello! Thanks for the Tatar translation. I'm not sure when I'll next work on Forge, but if I ever do, I'll include this translation as well. Have a great day!

Can You, please, copy this translation for Fabric too?

**Comment by TheCSDev** [comment]
Hello again! Sure thing, I can.

**Comment by TheCSDev** [comment]
By any chance, if you don't mind, could you translate the [whole json file](https://raw.githubusercontent.com/TheCSMods/mc-better-stats/main/betterstats-2-fabric-1.18.2/src/main/resources/assets/betterstats/lang/en_us.json) for the fabric version of the mod? It's bigger and has more stuff to be translated.
I tried translating other entries by myself, however I don't trust the machine translators, as they keep being inaccurate.

**Comment by Amirhan-Taipovjan-Greatest-I** [comment]
> By any chance, if you don't mind, could you translate the [whole json file](https://raw.githubusercontent.com/TheCSMods/mc-better-stats/main/betterstats-2-fabric-1.18.2/src/main/resources/assets/betterstats/lang/en_us.json) for the fabric version of the mod? It's bigger and has more stuff to be translated. I tried translating other entries by myself, however I don't trust the machine translators, as they keep being inaccurate.

Ok, but I need context for `Enable 'betterstats' communication with the server`

**Comment by Amirhan-Taipovjan-Greatest-I** [comment]
> Sure. It all started with [this beta release](https://legacy.curseforge.com/minecraft/mc-mods/better-stats/files/4501989), followed by [this release](https://legacy.curseforge.com/minecraft/mc-mods/better-stats/files/4510582), that is then followed by [this release](https://legacy.curseforge.com/minecraft/mc-mods/better-stats/files/4513777) and [yet another release](https://legacy.curseforge.com/minecraft/mc-mods/better-stats/files/4534016). You can read their changelogs at the attached hyperlinks for more info. Ik the changelogs are long, but they explain what is going on.
> 
> ### TLDR;
> The fabric version of this mod got some new features, features where the mod can optionally be installed on a fabric server, that then allows players to have their "statistics hud screen" be more accurate, as well as another feature that allows players to check each other's statistics.
> 
> If I remember correctly, I failed to port the "bss hud" feature to forge, so maybe you don't even know what that feature is. The "bss hud screen" allows people to "pin" statistics to their "GUI hud" aka screen, so they can see their stats in real-time as they play the game. By installing the mod on the server, the "bss hud" becomes more accurate/real-time.
> 
> The whole thing with `Enable 'betterstats' communication with the server` is just a privacy focused feature. You see, when the client mod communicates with the server mod, the server finds out the client has mods installed, including `betterstats`. In some cases, the player might not want that, which is why there's a feature where the player can toggle this "connection".
> 
> ### -
> Hopefully this clears things up a bit.

Please, check out #53.
